packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3026,6 +3026,12 @@ export interface IncomingCallState {
     startTime: Date;
 }
 
+// @beta
+export interface InlineImage {
+    imgAttrs: React_2.ImgHTMLAttributes<HTMLImageElement>;
```

Comment:
I'm not sure how I feel about this type. I'm not sure how easy it would be for developers to use. Should we instead make our own type? Or just make it a hashmap?

Reply:
> I'm not sure how I feel about this type. I'm not sure how easy it would be for developers to use. Should we instead make our own type? Or just make it a hashmap?

Is the concern more about the React_2 part or more about ImgHTMLAttributes?
For the ImgHTMLAttributes, it's a pretty straight forward API contains all the possible attribues for a image tag, reference is here https://use-form.netlify.app/interfaces/_node_modules__types_react_index_d_.react.imghtmlattributes.html

I think this is better than a hashmap so we're not limiting what type of attribues are supported